curriculum/challenges/chinese/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-8
 
 # --description--
 
-HTML <dfn>属性</dfn>是写在一个元素的开始标签中的特殊词汇，它们用于控制这个元素的行为。 在 `img` 元素中的 `src` 属性明确了一个图片的 URL（图片所在的位置）。 这是一个使用了 `src` 属性的 `img` 元素：`<img src="https://www.your-image-source.com/your-image.jpg">`。
+HTML <dfn>属性</dfn>是写在一个元素的开始标签中的特殊词汇，它们用于控制这个元素的行为。 在 `img` 元素中的 `src` 属性明确了一个图片的 URL（图片所在的位置）。 这是一个使用了 `src` 属性的 `img` 元素：`<img src="https://www.example.com/the-image.jpg">`。
```

Comment:
please remove this change, do not change the link here too
translated files are managed by our automatic pipelines

Reply:
Sorry I just don't know how to  remove the commit. Can you explain how to do it right?